### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by maxjerin**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/maxjerin/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1 @@
-# ReviewNinja-Welcome
\ No newline at end of file
```
Comment (maxjerin) [inline ReadMe.md]: Real conversations

Comment (maxjerin) [inline ReadMe.md]: !fixed

Comment (maxjerin) [inline ReadMe.md]: !fix

Comment (maxjerin) [inline ReadMe.md]: !fixed

**Pull request merged**